cmd/kubelet/app/server.go

```diff
@@ -192,7 +190,8 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 
 			// load kubelet config file, if provided
 			if configFile := kubeletFlags.KubeletConfigFile; len(configFile) > 0 {
-				kubeletConfig, err = loadConfigFile(configFile)
+				config, err := loadConfigFile(configFile)
```

Comment:
Can you please explain more about how:

```go
config, err := loadConfigFile(configFile)
*kubeletConfig = *config
```
is different than
```go
kubeletConfig, err = loadConfigFile
```

Reply:
`kubeletConfig, err = loadConfigFile` change the point of `kubeletConfig`

Reply:
Hmm, I'm afraid I'm still not following. Perhaps @yujuhong or @yifan-gu can weigh in?